src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteProgressCmdlet.cs

```diff
@@ -142,6 +142,16 @@ public SwitchParameter Completed
         [Parameter]
         public int SourceId { get; set; }
 
+        /// <summary>
+        ///
+        /// Character used to display progress.
+        ///
+        /// </summary>
+        /// <value></value>
+
+        [Parameter]
+        public char ProgressCharacter { get; set; } = 'o';
```

Comment:
If we already have the default in ProgressRecord.cs (`private char _progressCharacter = 'o'; `) so it should be removed here.